### PR TITLE
SDK-133 Create test coverage for the class clients.symphony.api.HealthcheckClient

### PR DESCRIPTION
**Pull request by Haiddi-Avencod**
clients.symphony.api.HealthcheckClient

### Ticket
SDK-133(https://perzoinc.atlassian.net/browse/SDK-133?atlOrigin=eyJpIjoiOWQ1ZTM1MGIzMzlkNDhhZGIzZmY2ZjY2MWQzOGU2MGEiLCJwIjoiaiJ9)

### Description
Test HealthcheckClient almost terminated. However in the method **performHealthCheckSuccessWithUnknownField()** I can't check the field **unknownField** because in the response object, there is no getter for this field.

### CODE REVIEW

File: symphony-bdk-legacy/symphony-api-client-java/src/test/java/it/clients/symphony/api/HealthcheckClientTest.java [hunk 1]
```diff
@@ -54,9 +61,16 @@ public void performHealthCheckSuccessWithUnknownField() {
...
-    assertEquals(true, response.getPodConnectivity());
+
+    assertTrue(response.getPodConnectivity());
+    assertTrue(response.getKeyManagerConnectivity());
+    assertTrue(response.getEncryptDecryptSuccess());
+    assertEquals("1.54.1", response.getPodVersion());
+    assertEquals("2.54.0", response.getAgentVersion());
+    assertTrue(response.getAgentServiceUser());
+    assertTrue(response.getCeServiceUser());
+    //assertEquals("a value", response.get);
```
Comment (symphony-elias) [inline symphony-bdk-legacy/symphony-api-client-java/src/test/java/it/clients/symphony/api/HealthcheckClientTest.java:74]: Please remove this commented line if useles.

Comment (Haiddi-Avencod) [reply symphony-bdk-legacy/symphony-api-client-java/src/test/java/it/clients/symphony/api/HealthcheckClientTest.java:74]: Actually I don't know if this line will be usefull or not until someone answer my question about the field **unknownField**

Comment (symphony-thibault) [reply symphony-bdk-legacy/symphony-api-client-java/src/test/java/it/clients/symphony/api/HealthcheckClientTest.java:74]: @Haiddi-Avencod You can ignore `unknownField`

Comment (Haiddi-Avencod) [reply symphony-bdk-legacy/symphony-api-client-java/src/test/java/it/clients/symphony/api/HealthcheckClientTest.java:74]: Ok, so should I remove this method `performHealthCheckSuccessWithUnknownField()` ?

**Review by symphony-thibault — APPROVED** [review]
LGTM 👍

**Pull request merged**